### ISSUE TITLE
Web Dev Experiments: Screen Reader Edition

### ISSUE THREAD

**Issue by circleofconfusion** [charmCityJs/talks#228]
**Talk Title:** 

Web Dev Experiments: Screen Reader Edition

**Talk Idea, described briefly:**

Rather than hectoring developers about how they should be making everything accessible, let's instead take an experimental approach to learning. I'll use an actual screen reader to demonstrate the drastic effects of simple HTML markup tweaks.

**Your name:** 

Shane Knudsen

**CCJS Slack username:**

ShaneKnudsen

**Twitter username (optional):** 

@CrclOfConfusion

**Estimated Talk Length: (1-15 min please):** 

15

**Availability:**

- [x] June 3rd, 2020
- [x] July 1st, 2020
- [ ] August 5th, 2020
- [x] September 2nd, 2020
- [x] October 7th, 2020
- [x] November 4th, 2020
- [x] December 2nd, 2020
